### ISSUE TITLE
Release to Release upgrade instructions needed for the users how to upgrade plugins and images to latest release.

### ISSUE THREAD

**Issue by jagraj** [eclipse/codewind-docs#113]
We need to document for the users how to upgrade plugins and get latest images for the future releases.

1. VS Code plugin
2. Eclipse plugin
3. Eclipse Che 
4. OpenAPI

**Comment by sishida** [comment]
I will work with Jag to document this content.

**Comment by johnmcollier** [comment]
Eclipse Che:
- Restart the Che workspace. Che will automatically pull down the newest 0.3.x versions of Codewind and the Theia extension.

**Comment by sujeilyfonseca** [comment]
<p align="center"><b>UPGRADE CODEWIND IMAGES TO LATEST VERSION IN VS CODE</b></p>

Latest Codewind plugins for VS Code are upgradeable, which means that users can update their existing installation without having to uninstall it.

1.	Go to _Extensions_ view in VS Code.

2.	Search for Codewind 0.3.X and proceed to install it. Latest Codewind images can also be added by selecting "Install from VSIX" and choosing the .vsix file corresponding to the 0.3.X images.

3.	Reload Codewind.

4.	Go to _Explorer_ view in VS Code.Reload Codewind.

- If images from previous versions/releases are installed on the system, but Codewind is not running, a pop-up window will ask you to install the most recent images (Figure 1).

<p align="center"><img width="419" alt="Screen Shot 2019-09-05 at 3 10 02 PM" src="https://user-images.githubusercontent.com/49531960/64376828-2704ab80-cff7-11e9-9c90-937872285d80.png"></br><b>Figure 1:</b> Codewind installation with existing images on the system</p>

- If an older version of Codewind is installed and running, a pop-up window will appear asking to update the existing outdated version. Codewind will stop automatically, and new images will begin to download (Figure 2).

<p align="center"><img width="416" alt="Screen Shot 2019-09-05 at 3 13 25 PM" src="https://user-images.githubusercontent.com/49531960/64376836-2a983280-cff7-11e9-8d12-b9c0b60fda14.png"></br><b>Figure 2:</b> Codewind installation with an existing version running on the system</p>

5.	Wait for the Codewind installation to complete, as shown in Figure 3.

<p align="center"><img width="448" alt="Screen Shot 2019-09-05 at 3 32 12 PM" src="https://user-images.githubusercontent.com/49531960/64376841-2e2bb980-cff7-11e9-94eb-1e53096fa4c1.png"></br><b>Figure 3:</b> Successful Codewind upgrade</p>

6.	After that, Codewind will start and be ready to use it.

**Comment by jagraj** [comment]
# Codewind Eclipse plugin upgrade

Codewind eclipse plugin can be upgraded to latest version of the plugin and plugin will automatically upgrade codewind images to latest version. 

1. Go to “Help->About Eclipse IDE” 

2. Click on “Installation details” 

![About-Eclipse-IDE](https://user-images.githubusercontent.com/713849/64385525-9cc34400-d004-11e9-8fb4-611246dfddf5.JPG)

3. Highlight “Codewind  tech preview” and Click on “Update” to look for latest release. 

![Codewind-Update](https://user-images.githubusercontent.com/713849/64385575-c5e3d480-d004-11e9-86b0-b281f79152a3.JPG)

4. Select latest version and click on “Next”. 

![Codewind-Update-latest](https://user-images.githubusercontent.com/713849/64385629-f1ff5580-d004-11e9-91b1-05664c6aa63e.JPG)

5. Review license and click on “Finish” 

![codewind-license](https://user-images.githubusercontent.com/713849/64385885-a305f000-d005-11e9-9edd-11b31eebb4d2.JPG)

6. Click on “Restart Now” to refresh Codewind to latest version. 

![eclipse-restart](https://user-images.githubusercontent.com/713849/64385928-c0d35500-d005-11e9-8713-077f98ee6c0f.JPG)

7. After restart navigate to “Codewind Explorer” view. Double click on “Codewind” to upgrade to latest version. 

![Codewind-upgrade](https://user-images.githubusercontent.com/713849/64385972-e4969b00-d005-11e9-8e24-aebdf712a6c8.JPG)

8. Following dialog window will be prompted to the user and click “OK”. 

![codewind-images-dialog](https://user-images.githubusercontent.com/713849/64386100-3f2ff700-d006-11e9-9a4b-5bf1f2778dc9.JPG)


9. After successful upgrade you should see following “Codewind Explorer” view with your projects.

![AfterUpgrade](https://user-images.githubusercontent.com/713849/64386185-70102c00-d006-11e9-9375-58ab68647445.JPG)

**Comment by sishida** [comment]
I have created a PR to add updating sections for Eclipse Che, VS Code, and Eclipse tools.

I have not added screenshots because it will take some time to download the screenshots, upload them to GitHub, and then adjust and test to make sure that they appear in the correct size. I might add them later after discussing with the rest of the ID team on how we want to handle screenshots going forward. In the past, we have tried to keep them to a minimum because they can become out of date and be time consuming to keep current.
